app/services/rapid_ready_for_decision/views/hypertension.erb

```diff
@@ -0,0 +1,5 @@
+<%= partial 'hypertension/intro' %>
+<%= partial 'hypertension/blood_pressure_readings' %>
+<%= partial 'hypertension/rating_schedule' %>
```

Comment:
So, in order to enforce some encapsulation, maybe this partiail helper should accept arguments explicitly so that these partials don't have access to everything

Reply:
Opinion: I don't think you need to at this point since they all pertain to hypertension.
Passing in arguments to `shared/medications` on line 4 might be a good idea to reduce assumptions about what variables `shared/medications` has access to.